### PR TITLE
Another shot at Terra docs

### PR DESCRIPTION
**Pull request by nonhermitian**
### Summary
Another shot at moving away from autodoc, and making things look good.

### Details and comments
@mtreinish can you please check the deploy script to verify that when this gets merged it does what it should.

### CODE REVIEW

**Comment by jaygambetta** [comment]
you need to also fix the links in the_elements.rst and I think there are some in the Readme.md and contributing.md as well.

File: docs/contributing_to_qiskit.rst [hunk 1]
```diff
@@ -295,21 +295,21 @@ universally depending on operating system.
...
 
          .. code:: sh
 
-            sudo apt install build-essential
+            dnf install @development-tools
 
       4. Install OpenBLAS development headers.
 
-         If you're using Ubuntu>=16.04 or an equivalent Debian Linux distribution,
+         If you're using Fedora or an equivalent Debian Linux distribution,
```
Comment (mtreinish) [inline docs/contributing_to_qiskit.rst]: Small nit: we probably should remove the debian reference here

```suggestion
         If you're using Fedora or an equivalent Linux distribution,
```

Comment (nonhermitian) [reply docs/contributing_to_qiskit.rst]: ahh missed one

File: docs/conf.py [hunk 2]
```diff
@@ -57,6 +57,7 @@
     'sphinx.ext.viewcode',
     'sphinx.ext.extlinks',
     'sphinx_tabs.tabs',
+    'sphinx_automodapi.automodapi'
```
Comment (mtreinish) [inline docs/conf.py]:
```suggestion
    'sphinx_automodapi.automodapi',
```

**Comment by mtreinish** [comment]
With the inline suggestion above I think this will work as expected. I locally tested the deploy script from a fresh venv (which is similar to a clean travis env) and ran it like travis would and it generated this commit:

https://github.com/mtreinish/qiskit.org/commit/0a3076d8a98f44146284990e32fa6c7773b069f7

which looks like it includes the updated api docs for terra

**Comment by nonhermitian** [comment]
Change integrated.

**Review by mtreinish — APPROVED** [review]
I think this should be good to go now. If there are any issues with the deploy script I wasn't able to find them locally and we can debug them post merge.

**Pull request merged**